packages/terser/src/index.js

```diff
@@ -43,31 +57,91 @@ function terserDirectory(input) {
     fs.mkdirSync(output);
   }
 
+  let work = [];
+  let active = 0;
+  let errors = [];
+
+  function exec([inputFile, outputFile]) {
+    active++;
+    let args = [TERSER_BINARY, inputFile, '--output', outputFile, ...residual];
+
+    spawn(process.execPath, args)
+        .then(
+            (data) => {
+              if (data.code) {
+                errors.push(inputFile)
+                // NOTE: Even though a terser process has errored we continue here to collect all of
+                // the errors. this behavior is another candidate for user configuration because
+                // there is value in stopping at the first error in some use cases.
+
+                log_error(
+                    'errored: ' + inputFile + '\n', 'OUT: ' + data.out + '\n',
```

Comment:
more readable with a template literal (backticks plus interpolation)

Reply:
i agree. 
Template literals get really messy if you every need to have white space for formatting that doesn't impact the output error message. Like in this case when the error message makes the line length too long and you need to format it for readability again the other way. You have to use multiple concatenated template literals for the middle ground i guess